components/filter/filter.js

```diff
@@ -617,6 +648,9 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 					` : nothing}
 				</div>
 			</d2l-list-item>
+			${item.additionalContent && item.selected && this._displayKeyboardTooltip
+		? html`<d2l-tooltip align="start" announced for="${itemId}" for-type="descriptor" @d2l-tooltip-hide="${this._handleTooltipHide}">${this.localizeHTML('components.filter.additionalContentTooltip')}</d2l-tooltip>`
```

Comment:
I was a little worried we'd have to use `announced` to do this. Unfortunately, avoiding it means baking tooltip into various list-item APIs. 😞